src/services/ArticleService.js

```diff
@@ -51,6 +51,27 @@ export class ArticleService {
 		return [];
 	}
 
+	async getAllArticlesByUser() {
+		try {
+			// TODO: this needs to be changed to an endpoint that just returns my articles, needs developing though
+			const { data } = await axios.get(`${backendAPI}/${this.BASE_URL}`);
```

Comment:
This currently just pulls in all articles, we need a new endpoint to be able to pull all a user's articles.

See: [website-backend#85](https://github.com/codesupport/website-backend/issues/85)